ArduPlane/Parameters.cpp

```diff
@@ -668,6 +668,13 @@ const AP_Param::Info Plane::var_info[] = {
     // @User: Standard
     GSCALAR(FBWB_min_altitude_cm,   "ALT_HOLD_FBWCM", ALT_HOLD_FBW_CM),
 
+    // @Param: ALT_FBWCM_MAX
+    // @DisplayName: Maximum altitude for FBWB and CRUISE modes
+    // @Description: This is the maximum altitude in meters that FBWB and CRUISE modes will allow. If you attempt to ascend above this altitude the plane will hold level. A value of zero means no limit. Must be at least 10 meters above ALT_HOLD_FBWCM.
+    // @Units: m
+    // @User: Standard
+    GSCALAR(fbwb_max_altitude_m,   "ALT_FBWCM_MAX", 0),
```

Comment:
need to document what frame (terrain, or above home) this is in
also, param name needs to not have 'CM' in it as the param is in meters
need to test behaviour with and without terrain